src/gurt/heap.c

```diff
@@ -1,24 +1,7 @@
 /*
  * (C) Copyright 2011,2016-2020 Intel Corporation.
```

Comment:
2021?

Reply:
I think the util/githooks/pre-commit is not handling this pattern (year1, year2-year3), may  need enhance it later, but will manually handle the ones found here for now to avoid massive re-processing all files in this PR for now, and update the script and do a final full checking to all DAOs files in a coming PR.